src/course-unit/breadcrumbs/Breadcrumbs.test.jsx

```diff
@@ -80,11 +114,13 @@ describe('<Breadcrumbs />', () => {
     const button = getByText(breadcrumbsExpected.section.displayName);
     userEvent.click(button);
     await waitFor(() => {
-      expect(queryAllByTestId('breadcrumbs-section-dropdown-item')).toHaveLength(5);
+      expect(queryAllByTestId('breadcrumbs-dropdown-item-level0')).toHaveLength(5);
     });
 
     userEvent.click(getByText(breadcrumbsExpected.subsection.displayName));
-    expect(queryAllByTestId('breadcrumbs-subsection-dropdown-item')).toHaveLength(2);
+    await waitFor(() => {
+      expect(queryAllByTestId('breadcrumbs-dropdown-item-level1')).toHaveLength(2);
```

Comment:
[optional]: More readable

```suggestion
      expect(queryAllByTestId('breadcrumbs-dropdown-item-level-1')).toHaveLength(2);
```

Reply:
Changed